hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConfigKeys.java

```diff
@@ -563,6 +563,10 @@ public final class OzoneConfigKeys {
       "ozone.om.keyname.character.check.enabled";
   public static final boolean OZONE_OM_KEYNAME_CHARACTER_CHECK_ENABLED_DEFAULT =
       false;
+  public static final String HDDS_DN_CHUNK_DATA_CHECK_ENABLED_KEY =
+      "hdds.datanode.chunk.data.validation.check";
+  public static final boolean HDDS_DN_CHUNK_DATA_CHECK_ENABLED_DEFAULT =
+      false;
```

Comment:
This seems to be used only in the test.  Also, config test is failing since same key is missing from XML.

https://github.com/Cyrill/ozone/actions/runs/8340503688/job/22825279215#step:6:196

Reply:
If this is an existing property from `DatanodeConfiguration`, then it shouldn't be added to `ozone-default.xml`, nor should it have a constant in `OzoneConfigKeys`.

The test can set it similar to:

https://github.com/apache/ozone/blob/d70d247a494a69cad28e96d003b25ae4856b8641/hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/TestBlockDeletingService.java#L662

https://github.com/apache/ozone/blob/d70d247a494a69cad28e96d003b25ae4856b8641/hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/TestBlockDeletingService.java#L665

Reply:
Thanks!